nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -607,10 +608,11 @@ private Description checkParamOverriding(
         } else {
           errorTree = getTreesInstance(state).getTree(paramSymbol);
         }
+
+        ErrorMessage errorMessage = new ErrorMessage(MessageTypes.WRONG_OVERRIDE_PARAM, message);
+        handler.suggest(state, paramSymbol, errorMessage);
```

Comment:
It seems we are calling `handler.suggest()` right before `createErrorDescription()` a lot.  Could we change the parameters of `createErrorDescription()` such that we call `handler.suggest()` inside it?

Reply:
If we can do the above, it's not even clear to me we need a `handler.suggest()` method, since the only call would be coming from `createErrorDescription()` and it's less critical to me that only core logic appear in that method

Reply:
I removed the ```suggest handler```. Everything is implemented in ```ErrorBuilder``` class now. Please find it here 5207bf53a421d069c6cc90f58c2eda7ea5e43c45. I think we may need to rename ```CreateErrorDescription``` method and maybe even ```ErrorBuilder``` since now it creates many different sorts of object not just errors.